src/types/models.ts

```diff
@@ -32,11 +32,31 @@ export const NOISE_SENSOR_DEVICE_TYPES = [
   "minut_sensor",
 ]
 export type NoiseSensorDeviceType = typeof NOISE_SENSOR_DEVICE_TYPES[number]
+export type NoiseSensorDeviceProperties = CommonDeviceProperties
 
-export const THERMOSTAT_DEVICE_TYPES = ["nest_thermostat"]
+export const THERMOSTAT_DEVICE_TYPES = ["nest_thermostat", "ecobee_thermostat"]
 export type ThermostatDeviceType = typeof THERMOSTAT_DEVICE_TYPES[number]
-
-export type NoiseSensorDeviceProperties = CommonDeviceProperties
+export type ThermostatDeviceProperties = CommonDeviceProperties & {
+  is_cooling: boolean
```

Comment:
I am not sure if all of these thermostat properties are necessary here. The properties for noise sensors are just `CommonDeviceProperties`: `export type NoiseSensorDeviceProperties = CommonDeviceProperties`